src/component/playground/Playground/Playground.tsx

```diff
@@ -13,37 +14,112 @@ import { PlaygroundConnectionFieldset } from './PlaygroundConnectionFieldset/Pla
 import { PlaygroundCodeFieldset } from './PlaygroundCodeFieldset/PlaygroundCodeFieldset';
 import useToast from 'hooks/useToast';
 import { formatUnknownError } from 'utils/formatUnknownError';
+import { PlaygroundResultsTable } from './PlaygroundResultsTable/PlaygroundResultsTable';
+import { ContextBanner } from './PlaygroundResultsTable/ContextBanner/ContextBanner';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import usePlaygroundApi from 'hooks/api/actions/usePlayground/usePlayground';
+import { PlaygroundResponseSchema } from 'hooks/api/actions/usePlayground/playground.model';
 
 interface IPlaygroundProps {}
 
 export const Playground: VFC<IPlaygroundProps> = () => {
     const theme = useTheme();
+    const [environment, onSetEnvironment] = useState<string>('');
+    const [projects, onSetProjects] = useState<string[]>([]);
     const [context, setContext] = useState<string>();
-    const [contextObject, setContextObject] = useState<string>();
+    const [loading, setLoading] = useState(false);
```

Comment:
Could we use the `loading` property that comes from the `usePlaygroundApi`. We get that state for free and it's connected to the API call.